packages/artifactory-jcr/generated-changes/dependencies/artifactory/dependency.yaml

```diff
@@ -0,0 +1,2 @@
+workingDir: ""
+url: https://charts.jfrog.io/artifactory/api/helm/jfrog-charts/artifactory-11.7.4.tgz
```

Comment:
FYI the partner will also need to manually keep this dependency file up-to-date. It does not automatically get modified on rebasing the upstream.

Does this need to be called out in the docs or announced somewhere else?

Reply:
I think we can add this to the docs for charts-build-scripts v0.2.1
It has happened once for Fleet that the dependency was not changed to the latest when the Fleet itself was changed to the latest.

Reply:
I will add it to the docs as part of the steps for upgrading.